FreeRTOS/FreeRTOS
# Posix Full Demo exit on error

Posix Full Demo exit on error

Description
-----------
When the Posix demo encounters an error, it will exit with 1
if ctrl_C is pressed it will exit with 2
make [USER_DEMO=BLINKY_DEMO|FULL_DEMO] will build the appropriate demo

Adding this feature is to allow the demo to be ran on a nightly job

Test Steps
-----------
Run the Demo


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
